rootfs/api/serializers.py

```diff
@@ -252,6 +307,21 @@ def validate_registry(self, data):
 
         return data
 
+    def validate_healthcheck(self, data):
+        for key, value in data.items():
+            if value is None:
+                raise serializers.ValidationError("Healthcheck value for " + key + "can't be None")
+
+            if key not in ['livenessProbe', 'readinessProbe']:
+                raise serializers.ValidationError(
+                    "Healthcheck keys must be either livenessProbe or readinessProbe")
+            try:
+                jsonschema.validate(value, PROBE_SCHEMA)
+            except jsonschema.ValidationError as e:
```

Comment:
Do you have any tests to trigger a validation? codecov.io says this line was never hit